liquid-integration/src/main/java/com/e/liquid_integration/data/LiquidClient.kt

```diff
@@ -0,0 +1,263 @@
+                    if (response.isSuccessful) {
+                        val userId = response.body()?.payload?.liquidUserId
+                        if (!TextUtils.isEmpty(userId)) {
+                            storeUserId(userId!!)
+                            callback.onSuccess(userId)
+                        } else {
+                            callback.onError(LiquidException("Error obtaining liquid user id", response.message(), response.code()))
+                        }
+                    } else {
+                        callback.onError(LiquidException("Error obtaining liquid user id", response.message(), response.code()))
+                    }
+                }
+
+                override fun onFailure(call: Call<UserKycState?>, t: Throwable) {
+                    callback.onError(Exception(t))
+                }
+            })
+
+        } catch (e: Exception) {
+            callback.onError(e)
+        }
+    }
+
+
+    fun getUserAccountBalance(sessionId: String, callback: Callback<String>) {
+        try {
+            val token = getAuthToken("/api/v1/session/$sessionId/accounts")
+
+            service.getUserAccount(sessionId, token).enqueue(object : retrofit2.Callback<String?> {
+
+                override fun onResponse(call: Call<String?>, response: Response<String?>) {
+                    if (response.isSuccessful) {
+                        callback.onSuccess(response.body()!!)
+                    } else {
+                        callback.onError(LiquidException("Error obtaining liquid user id", response.message(), response.code()))
+                    }
+                }
+
+
+                override fun onFailure(call: Call<String?>, t: Throwable) {
+                    callback.onError(Exception(t))
+                }
+            })
+
+        } catch (e: Exception) {
+            callback.onError(e)
+        }
+    }
+
+
+    fun revokeAccessToken(callback: Callback<String?>) {
+        try {
+
+            val token = getAuthToken("/api/v1/session/$storedSessionId/terminate")
+
+            service.terminateSession(storedSessionId!!, token).enqueue(object : retrofit2.Callback<LiquidTerminateSession?> {
+
+                override fun onResponse(call: Call<LiquidTerminateSession?>, response: Response<LiquidTerminateSession?>) {
+                    if (response.isSuccessful) {
+                        callback.onSuccess("")
+                    } else {
+                        callback.onError(LiquidException("Error in terminate session", response.message(), response.code()))
+                    }
+                }
+
+                override fun onFailure(call: Call<LiquidTerminateSession?>, t: Throwable) {
+                    callback.onError(Exception(t))
+                }
+            })
+        } catch (e: Exception) {
+            callback.onError(e)
+            Log.v("onError....", e.toString())
```

Comment:
how are errors and success statuses log?  This is important for tech support.

Reply:
Removed this log